boards/seeed/xiao_esp32s3/xiao_esp32s3_procpu_sense.yaml

```diff
@@ -0,0 +1,22 @@
+identifier: xiao_esp32s3/esp32s3/procpu/sense
+name: XIAO ESP32S3 PROCPU
```

Comment:
```suggestion
name: XIAO ESP32S3 Sense PROCPU
```

Reply:
Done, also fixed doc build. Now it should build fine.